Cosmetics.py

```diff
@@ -705,6 +705,10 @@ def patch_button_colors(rom, settings, log, symbols):
 
             log_dict['colors'][patch] = color_to_hex(colors[patch])
 
+    if settings.correct_model_colors:
+        symbol = rom.sym('CORRECT_MODEL_COLORS')
+        rom.write_byte(symbol, 0x01)
+
```

Comment:
You'll need to move this symbol into the cosmetic context in config.asm and increment the cosmetic version/add a new `patch_set` for the new cosmetic version. Otherwise, this will write garbage into the wrong location when using patch files of a differing version.